### ISSUE TITLE
Self hosting

### ISSUE THREAD

**Issue by Heavyrisk** [bbarke/brp-fpp-voting#12]
Will there ever be the option of self hosting?

**Comment by bbarke** [comment]
Yes I hope so. I made some poor decisions when I created the server/client and tied them together with the rest of my website - but I hope to decouple that, sanitize it, and open source the client/server under a license (like mit or apache) that'll allow people to do self hosting.
